openapi.yaml

```diff
@@ -16,60 +16,399 @@ host: api.oregonstate.edu
 basePath: /v1
 externalDocs:
   description: GitHub Repository
-  url: https://github.com/osu-mist/express-api-skeleton
+  url: https://github.com/osu-mist/poker-api
 produces:
   - application/json
 paths:
-  /pets:
-    get:
-      summary: List all pets
+  /games:
+    post:
+      summary: Create a poker game
       tags:
-        - pets
-      description: Get all pets
-      operationId: getPets
+        - games
+      consumes:
+        - application/json
+      operationId: postGame
       parameters:
         - $ref: '#/parameters/authorization'
-        - $ref: '#/parameters/pageNumber'
-        - $ref: '#/parameters/pageSize'
+        - in: body
+          name: requestBody
+          schema:
+            $ref: '#/definitions/GamePostBody'
+      responses:
+        '201':
+          description: Successfully create a game
+          schema:
+            $ref: '#/definitions/GameResult'
+          headers:
+            Location:
+              type: string
+              format: url
+              description: Location of the newly created game
+        '400':
+          description: Invalid game object
+          schema:
+            $ref: '#/definitions/Errors'
+        '500':
+          description: Internal server error
+          schema:
+            $ref: '#/definitions/Errors'
+    get:
+      summary: Info of a list of game
+      tags:
+        - games
+      operationId: getGames
+      parameters: 
+        - $ref: '#/parameters/authorization'
+        - in: query
+          name: round
+          type: string
+          description: The current round of the game
         - in: query
-          name: species
-          description: Search by pet species
-          required: false
+          name: memberId
           type: string
+          description: The memberId of members that are playing this game
```

Comment:
Since the `game` resource doesn't have a `memberIds` property, could you remove this query parameter?